app/models/core_induction_programme.rb

```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
 
 class CoreInductionProgramme < ApplicationRecord
+  has_many :course_years
```

Comment:
If we destroy Core Induction Programme - do we expect the associated records to be deleted as well?

https://stackoverflow.com/questions/29560805/how-to-use-dependent-destroy-in-rails

Reply:
Hmm, 'who knows' is the short answer. 

From a dev perspective, probably yes. 

From user perspective, I don't think we got to the point of figuring out whether we want to create / delete cips at all. But when we eventually do, I think this whole thing will change into some many-to-many relation. 

For time being I will add a cascade delete here. Good spot :)

Reply:
Sweet!